packages/saltcorn-data/models/view.ts

```diff
@@ -712,6 +713,10 @@ class View implements AbstractView {
   ): Promise<any> {
     const { getState } = require("../db/state");
     const state = getState();
+    // Hash password in body
+    if (body?.password) {
+      body.password = await hash(body.password, 10);
```

Comment:
this is not great. can we do without this?

Reply:
When a non-admin is adding a new user from a view, the password maybe kept but it is not hashed. For now I moved the logic to "runPost" in `/saltcorn-data/base-plugin/viewtemplates/edit.js`